terraform/aks/statuscake.tf

```diff
@@ -4,7 +4,6 @@ module "statuscake" {
   source = "./vendor/modules/aks//monitoring/statuscake"
 
   uptime_urls = compact(concat([module.web_application["main"].probe_url], local.statuscake_additional_hostnames))
-  ssl_urls    = compact(local.statuscake_additional_hostnames)
 
```

Comment:
do we need to remove the statuscake alert for prod?

Reply:
The shared excel have following   thing.

https://www.publish-teacher-training-courses.service.gov.uk/ping
https://www.find-postgraduate-teacher-training.service.gov.uk/ping